tests/integration/backend/test_backend.py

```diff
@@ -1,13 +1,17 @@
 # backend module integration tests
 
-
+# package imports
 from imars3d.backend.workflow.engine import WorkflowEngineAuto
 from imars3d.backend.workflow.engine import WorkflowEngineError
 from imars3d.backend.workflow.validate import JSONValidationError
 
+# third party imports
+import pytest
+
+# standard library imports
+from copy import deepcopy
 import json
 import numpy as np
-import pytest
 from pathlib import Path
 
 JSON_DATA_DIR = Path(__file__).parent.parent.parent / "data" / "json"
```

Comment:
There is a [fixture for](https://github.com/ornlneutronimaging/iMars3D/blob/next/tests/conftest.py#L6) this. Can you change to use it?

Reply:
:heavy_check_mark: